cpp/core.cc

```diff
@@ -256,6 +268,15 @@ int LastSignal() {
   return gSignalHandler->last_sig_num_;
 }
 
+int SigintCount() {
+  if (gSignalHandler == nullptr) {
```

Comment:
Comment about when it can be nullptr?  I guess it depends on the order of InitShell() vs. ReadByte() and so forth?

I am wondering if it should be an DCHECK(), but I guess we have not really settled on the order of those things ...

Reply:
Oh this is a holdover from a branch where I was trying to throw from `Alloc()`, which might have been allocating `gSignalHandler`. I'll change this to a `DCHECK`

Reply:
Hm should this be DCHECK or not?  Either way adding a comment would help

Reply:
Ah doh. Missed this one